_components/in-page-navigation/guidance/implementation.md

```diff
@@ -19,9 +19,9 @@ Implementing the in-page navigation component requires a few additions and adjus
 {:.usa-content-list }
 - **Root margin.** The root margin property (`data-root-margin`) is a string that allows you to define the observable area. This set of values grows or shrinks the observable area from each side of the root element's bounding box before computing intersections. It can accept values similar to the CSS margin property, and can be percentages. By default, the in-page navigation component is set to `0px 0px 0px 0px`.
 - **Threshold.** The threshold property (`data-threshold`) determines how much of the observable section must be present in the observed area before the nav's item is set to current. The default is `1`, which means 100% of the observables must be in the observed section. You can set `data-threshold` to any integer between 0 and 1.
```

Comment:
Since some of the other attributes get guidance, I think the `main-content-selector` could benefit from a section as well. I know it's in the changelog, but I like your summary note and I think users might benefit from it being included in the guidance section:

> This attribute allows users to designate which element they want the component to pull headers from.
      If `data-main-content-selector` is not defined, the component will pull from the `<main>` element.

Reply:
@mejiaj Yes, I pulled some of the content out of the changelog and into the main content region in [22b77f8](https://github.com/uswds/uswds-site/pull/2190/commits/22b77f81cdd07b831882b27feab730ba74f4870e) and [dbe4035](https://github.com/uswds/uswds-site/pull/2190/commits/dbe4035f7cb695070a563cbf190eb3a547f9c8e1).